test/e2e/awsiam_test.go

```diff
@@ -227,8 +257,8 @@ func TestSnowKubernetes121UbuntuAWSIamAuth(t *testing.T) {
 		framework.NewSnow(t, framework.WithSnowUbuntu121()),
 		framework.WithAWSIam(),
 		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube121)),
-		framework.WithEnvVar("SNOW_PROVIDER", "true"),
-		framework.WithEnvVar("FULL_LIFECYCLE_API", "true"),
```

Comment:
We don't need this env var anymore?

Reply:
we still need it, just using the features package to use the constant instead.